src/flag_gems/ops/embedding.py

```diff
@@ -0,0 +1,188 @@
+import logging
+import math
+
+import torch
+import triton
+import triton.language as tl
+
+from ..utils import libentry
+
+
+@libentry()
+@triton.jit
+def embedding_kernel(
+    Y,  # pointer to the output
+    X,  # pointer to the input
+    W,  # pointer to the weights
+    N: tl.constexpr,  # number of columns in X
+    BLOCK_SIZE: tl.constexpr,
+):
+    pid = tl.program_id(0)
+    Y += pid * N
+    X += pid
+
+    mask = tl.arange(0, BLOCK_SIZE) < N
+    cols = tl.arange(0, BLOCK_SIZE)
+
+    row_idx = tl.load(X).to(tl.int32)
+    W += row_idx * N
+    embedding_weight = tl.load(W + cols, mask, other=0.0)
+    tl.store(Y + cols, embedding_weight, mask)
+
+
+@libentry()
+@triton.jit
+def indice_freq_kernel(
+    indices_freq,  # indice frequency
```

Comment:
```suggestion
    indices_freq,
```
Also not an informative code comment here